avocado/utils/asset.py

```diff
@@ -194,6 +196,14 @@ def _check_file(path, filehash=None, algorithm=None):
             return False
 
     @staticmethod
+    def _clear_file_signatures(path):
+        for algo in crypto.algorithms_supported():
+            hashfile = '%s.%s' % (path, algo)
+            if os.path.isfile(hashfile):
+                log.debug('remove obsoleted hashfile %s' % hashfile)
+                os.unlink(hashfile)
+
+    @staticmethod
```

Comment:
Yes, this is also addressed by https://github.com/avocado-framework/avocado/pull/1325